config/default.yml

```diff
@@ -2951,6 +2951,11 @@ Style/DoubleNegation:
     - allowed_in_returns
     - forbidden
 
+Style/DuplicateRegexpCharacterClassElement:
+  Description: 'Checks for duplicate elements in Regexp character classes.'
+  Enabled: pending
+  VersionAdded: '0.94'
```

Comment:
After rebasing update this to `1.1`. :-)

Reply:
🎉 amazing - 1.0! Great news @bbatsov - thank you for all of your and the core team's hard work!